src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Consent.cshtml

```diff
@@ -0,0 +1,82 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.ConsentModel
+@{
+    ViewData["Title"] = "Permission to share details";
+}
+
+@section Back {
+    <a href="/ProfessionalReferral/ConnectFamilyToServiceStart?id=@Model.Id&name=@Model.Name" class="govuk-back-link">Back</a>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        @if (!Model.ValidationValid)
+        {
+            <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" data-module="govuk-error-summary">
+                <h2 class="govuk-error-summary__title" id="error-summary-title">
+                    There is a problem
+                </h2>
+                <div class="govuk-error-summary__body">
+                    <ul class="govuk-list govuk-error-summary__list">
+                        <li>
+                            <a href="#service-name" data-testid="enter-service-name-error">Select whether you have permission from the family to share details.</a>
```

Comment:
The href should contain the id of the radio input element, so that when the link is clicked, the page scrolls to the input with the error.
Double check, but I think it should be `#consent`.

Reply:
Fixed